src/test/java/BeforeClasses.java

```diff
@@ -0,0 +1,42 @@
+import api.*;
+import org.apache.http.HttpStatus;
+import org.testng.Assert;
+import org.testng.annotations.BeforeSuite;
+
+import static configuration.env.CONFIG;
+
+public class BeforeClasses {
+    public ApiRequest apiRequest;
+    public String token;
+    public String instance_url;
+    protected static final int STATUS_OK = 200;
```

Comment:
remove this static variables. there is enums that provide these codes like HttpStatus.SC_OK